README.md

```diff
@@ -22,105 +52,56 @@ cd ../../
 php bin/console pr:mo install fop_console
 ```
 
-## Current commands
-
-* `fop:clear-cache` Clear the cache folder super fast
-* `fop:debug-mode` Enable or Disable debug mode
-* `fop:shop-status` Display shop(s) status(es)
-* `fop:maintenance` get status or change maintenance mode, list or add maintenance ip address
-* `fop:images:generate:categories` Regenerate categories thumbnails
-* `fop:images:generate:manufacturers` Regenerate manufacturers thumbnails
-* `fop:images:generate:products` Regenerate products thumbnails
-* `fop:images:generate:stores` Regenerate stores thumbnails
-* `fop:images:generate:suppliers` Regenerate suppliers thumbnails
-* `fop:generate:htaccess` Generate the .htaccess file
-* `fop:generate:robots`   Generate the robots.txt file
-* `fop:theme-reset` Reset current (or selected) theme
-* `fop:add-hook` : Create a new hook in database
-* `fop:unhook-module` : Ungraft module on specific hook
-* `fop:hook-module` : Graft module on specific hook
-* `fop:latest-products`: Displays the latest products
-* `fop:export`: Exports object models in XML
-* `fop:check-container`   Health check of the Service Container, for now list the services we can't use in Symfony commands
-* `fop:configuration:export` Export configuration values to a json file
-* `fop:configuration:import` Import configuration values from a json file
-* `fop:customer-groups`: Move or add in bulk clients from one group client to another
-
-## Create your owns Commands
-
-The official documentation from PrestaShop and Symfony Core teams are still right, but you needs
-to extends our class.
-
-```php
-<?php
-
-// psr-4 autoloader
-
-// if the command is located at src/Commands
-namespace FOP\Console\Commands; 
-// or if command is located in a subfolder
-namespace FOP\Console\Commands\Domain; // e.g. namespace FOP\Console\Commands\Configuration
-
-use FOP\Console\Command;
-use Symfony\Component\Console\Style\SymfonyStyle;
-use Symfony\Component\Console\Input\InputInterface;
-use Symfony\Component\Console\Output\OutputInterface;
-
-final class MyCommand extends Command
-{
-    /**
-     * {@inheritdoc}
-     */
-    protected function configure()
-    {
-        $this
-            ->setName('fop:mycommand') // e.g 'fop:shop-status'
-            // or
-            ->setName('fop:domain:mycommand') // e.g 'fop:configuration:export' 
-            ->setDescription('Describe the command on a user perspective.');
-    }
-
-    /**
-     * {@inheritdoc}
-     */
-    protected function execute(InputInterface $input, OutputInterface $output)
-    {
-        $io = new SymfonyStyle($input, $output);
-        $io->text('Hello friends of PrestaShop!');
-
-        return 0; // return 0 on success or 1 on failure.        
-    }
-}
+## Getting started
+
+In a shell (call it shell, console or terminal), at the root of a Prestashop installation run the command, type this command to list all available commands.
```

Comment:
Encore une petite coquille à ce niveau je dirais ?
uniquement "type this command" plutôt ?